### DISCUSSION TITLE
Payback Rate Adjustments in editor

### DISCUSSION THREAD

**Discussion by worthlessnut9** [ppy/osu#24093] [Ideas]
in the editor make it so when you click on the "Payback Rate" text on the bottom right a pop up similar to the pop up for the current time of the beatmap will appear with the current playback rate and from there you can adjust the payback rate to smaller increments lower then 25%

**Comment by peppy** [comment]
Lower than 25% loses a lot of accuracy and is not too useful. I don't think we would allow it.

**Reply by worthlessnut9** [reply]
It feels it would help with mapping higher bpm maps for hard difficulty especially in mania. From personal experience in mapping many 300+ bpm maps its can still be difficult ass it feel its still fast to tell snapping

**Reply by peppy** [reply]
Bega the question of how you're supposed to play the map if you can't even map while listening at 1x speed...

Have you tried using the visual cues for snapping?

**Reply by worthlessnut9** [reply]
I don't think I understand what you mean, the songs I'm referring to are at about 320 to 375 bpm metal maps so it can be hard distinguishing the drums kicks especially when the inherent instruments are a bit louder

**Reply by ItsShamed** [reply]
With 50% of speed in this range of BPMs, you should definitely be able to map. I'd even argue that you don't even need lower than that.

> it can be hard distinguishing the drums kicks especially when the inherent instruments are a bit louder

Adding a speed lower than 25% will just worsen your case, as mentioned above.

Also, I don't see where the problem is, once you got the timing down for said maps can't you just use the editor grid to guess and snap your rhythm?

![image](https://github.com/ppy/osu/assets/39100084/c8259bcd-659d-42bd-9d91-4b1ab5a30d9e)

**Reply by worthlessnut9** [reply]
I don't think so, over the last few days I was trying diffrent settings and diffrent speeds and even diffrent audio devices and I can't find any good way to distinguish them with out guessing and assuming.

**Reply by ItsShamed** [reply]
Then the only culprit left is that your audio file may have a terrible quality. This is the last explanation viable to me.

**Reply by worthlessnut9** [reply]
I don't know if I can link the map for an example, I don't use github often.
